### PR TITLE
Revert "#13 Building the board with JavaFX"

### PR DESCRIPTION
**Pull request by emprestes**
Reverts emprestes/chessgame#42

## Building board (JavaFX)

**IMPORTANT**: if you repeat the logic more than once, it will be considerate a bad practice (bad smell).

### Class `chessgame.app.Chess`
- [x] Definition
- [x] Implementation
- [x] JavaDoc

### Interface `chessgame.app.fx.Controller`
- [x] Definition
- [x] JavaDoc

### Class `chessgame.app.fx.util.FXMLHelper`
- [x] Definition
- [x] Implementation
- [x] JavaDoc

### Abstract class `chessgame.app.fx.controller.AsbtractBoard`
- [x] Definition
- [x] Implementation
- [x] JavaDoc

### Class `chessgame.app.fx.controller.BoardController`
- [x] Definition
- [x] Implementation
- [x] JavaDoc

### FXML `chessgame.app.fx.view.BoardView`
- [x] Definition
- [x] Implementation

## Apache Maven
- [x] Include `exec-maven-plugin`

### CODE REVIEW

**Comment by emprestes** [comment]
@frenangomes  I finished it.

File: app-fx/src/main/resources/.gitignore [hunk 1]
```diff
@@ -1,28 +0,0 @@
-# Unix
```
Comment (frenangomes) [inline app-fx/src/main/resources/.gitignore]: Why do you is deleting this lines?

**Comment by frenangomes** [comment]
@emprestes  LGTM 👍 

I only did one question :)

File: app-fx/src/main/resources/.gitignore (hunk 1 above)
Comment (emprestes) [inline app-fx/src/main/resources/.gitignore]: I don't have idea. Please, ask Git on https://git-scm.com/community

**Pull request merged**